Source/AFError.swift

```diff
@@ -220,6 +220,10 @@ public enum AFError: Error {
     case sessionInvalidated(error: Error?)
     /// `URLSessionTask` completed with error.
     case sessionTaskFailed(error: Error)
+    /// An `OutputStream` returned an error during a write operation.
+    case streamError(Error?)
```

Comment:
We can't add error cases here until the next major version, FYI. For minor releases a new error type in the custom case could be used.

Reply:
And this may be something we need in deployment, so a separate PR for the change would probably be good.

Reply:
Yep, I have this pulled out in a separate branch and will put it in a PR shortly. To avoid making this a breaking issue I can avoid adding a new error case.